### ISSUE TITLE
python3 compatible DiViMe

### ISSUE THREAD

**Issue by junghanw** [srvk/DiViMe#68]
List of commands 
- [X] `tools/test.sh`
- [x] `tools/noisemes_sad.sh`
- [x] `tools/noisemes_full.sh`
- [x] `tools/opensmile_sad.sh`
- [x] `tools/tocombo_sad.sh`
- [ ] `tools/diartk.sh`
- [x] `tools/yunitate.sh`
- [ ] `tools/yuniSeg.sh`
- [ ] `tools/eval.sh`

List of issues 
- [x] model encoding problem (Should update all model/pickled files with unicode encoding)
      -  changed dill to python pickle to use encoding parameter when loading pickle files. 
- [x] Theano cython problem 
```
/opt/anaconda/envs/divime/lib/python3.6/site-packages/theano/scan_module/scan_perform_ext.py:76: UserWarning: The file scan_perform.c is not available. This donot happen normally. You are probably in a strangesetup. This mean Theano can not use the cython code for scan. If youwant to remove this warning, use the Theano flag'cxx=' (set to an empty string) to disable all ccode generation.
```

**Comment by junghanw** [comment]
Installing theano with pip instead of conda solves theano-cython problem

**Comment by junghanw** [comment]
closed as of #77
